sample_size/sample_size_calculator.py

```diff
@@ -82,12 +85,12 @@ def get_sample_size(self) -> float:
 
         return sample_size
 
-    def register_metric(self, metric_type: str, metric_metadata: Dict[str, float]) -> None:
+    def register_metric(self, metric_type: List[str], metric_metadata: List[Dict[str, float]]) -> None:
         VAR_REGISTER_FUNC_MAP = {
             "boolean": "register_bool_metric",
             "numeric": "register_numeric_metric",
             "ratio": "register_ratio_metric",
         }
-
-        register_func = getattr(self, VAR_REGISTER_FUNC_MAP[metric_type])
-        register_func(**metric_metadata)
+        for i in range(self.variants - 1):
```

Comment:
confused by this - metrics number !== number of variants right?

Reply:
Oh that's right, it should be `for i in range(len(metric_type))` instead. `variants` should be used only when `MultipleTesting` simulates p-values